nss-day-cohort-47/Class-Website
# Added Dad joke to footer, changed some of the variables to align with…

… JSON

## Link to Ticket

TITLE [Link]()

## Description of Proposed Changes

Added  Dad Joke to footer 
Also changed some of the variable names in the main.js forEach loop to align with cohort.JSON file better. Note commented out image lines 50 & 80 in main.js 


- 

## Steps to Test


Click button in footer to get Dad Joke. Joke is displayed


```sh
git fetch --all
git checkout CC-DadJoke

```


## Impacted Areas in Application

List general components of the application that this PR will affect:

index.html  footer
main.js added joke methods to bottom 

* 

## Mentions @username

Tag users that need to review this code
@pour-theoil 
@JSchugt 
@bvinson23 
@dudley-codes
